tests/ui/conftest.py

```diff
@@ -32,35 +42,100 @@ def jwt_issuer(base_url, variables):
 @pytest.fixture(scope='session')
 def jwt_secret(base_url, variables):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
+        hostname = urlparse.urlsplit(base_url).hostname
         return variables['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
-def jwt_token(base_url, jwt_issuer, jwt_secret):
-    payload = {
-        'iss': jwt_issuer,
-        'iat': datetime.datetime.utcnow(),
-        'exp': datetime.datetime.utcnow() + datetime.timedelta(seconds=30)}
-    return jwt.encode(payload, jwt_secret, algorithm='HS256')
+def initial_data(transactional_db):
+    from olympia.amo.tests import addon_factory, user_factory
+    from olympia.addons.models import AddonUser
+
+    for x in range(10):
+        AddonUser.objects.create(user=user_factory(), addon=addon_factory())
```

Comment:
Just noticed it, if you ever have the need to use it, there's also `olympia.landfill.user:generate_addon_user_and_category` which does this and creates a proper (featured) category too.